spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -63,4 +63,9 @@ private SparkSQLProperties() {}
   // Controls the WAP branch used for write-audit-publish workflow.
   // When set, new snapshots will be committed to this branch.
   public static final String WAP_BRANCH = "spark.wap.branch";
+
+  // Controls whether to push down Iceberg system function
+  public static final String SYSTEM_FUNC_PUSH_DOWN_ENABLED =
+      "spark.sql.iceberg.system-function-push-down.enabled";
+  public static final boolean SYSTEM_FUNC_PUSH_DOWN_ENABLED_DEFAULT = false;
```

Comment:
Why is there a setting for this and why does it default to false?

Iceberg always has the option of rejecting functions by returning them rather than consuming them. I don't see a reason to have a flag for this.

Reply:
Compared with `StaticInvoke`, `ApplyFunctionExpression` may have a little decrease in performance because it can not leverage codegen.